pyomo/solvers/plugins/solvers/XPRESS.py

```diff
@@ -35,421 +15,19 @@ def __new__(cls, *args, **kwds):
         try:
             mode = kwds['solver_io']
             if mode is None:
-                mode = 'lp'
+                mode = 'python'
             del kwds['solver_io']
         except KeyError:
-            mode = 'lp'
+            mode = 'python'
```

Comment:
Would the following would be cleaner / clearer?
```python
mode = kwds.pop('solver_io', 'python')
if mode is None:
    mode = 'python'
```

Reply:
Thanks @jsiirola. Great change.